<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DomeSimulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="params">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>235</width>
       <height>280</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="dome_status_label">
          <property name="text">
           <string>Dome Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dome_status_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>STOPPED</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OFF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HOMING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MOVING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>STOWED</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SUNTRACK</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UNKNOWN</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="home_status_label">
          <property name="text">
           <string>Home Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="home_status_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>READY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NOT_READY</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="shutter_status_label">
          <property name="text">
           <string>Shutter Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="shutter_status_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>CLOSED</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OPEN</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OPENING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CLOSING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>STOPPED</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UNKNOWN</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="control_status_label">
          <property name="text">
           <string>Control Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="control_status_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>AVAILABLE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MANUAL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CONSOLE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>REMOTE</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="close_status_label">
          <property name="text">
           <string>Close Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="close_status_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>READY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NOT_READY</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="weather_status_label">
          <property name="text">
           <string>Weather Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="weather_status_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>READY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NOT_READY</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="sunlight_status_label">
          <property name="text">
           <string>Sunlilght Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sunlight_status_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>READY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NOT_READY</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="wetness_status_label">
          <property name="text">
           <string>Wetness Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="wetness_status_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>READY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NOT_READY</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="current_status_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>531</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="update_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>520</y>
      <width>113</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="dome_params_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dome Parameters</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="weather_params_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Weather Parameters</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="weather_params_label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Faults</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
